Products/BeakerSessionDataManager/sessiondata.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 
 from AccessControl.SecurityInfo import ClassSecurityInfo
+from AccessControl.Permissions import access_contents_information
```

Comment:
@pgrunewald Does this one need to be made conditional too?

Reply:
No, I dont think so. It has been around [2010](https://github.com/zopefoundation/AccessControl/commit/635bd537ab0eacbba9b6a6d2596a8490e16b7203#diff-68be8e0e35554a48f0906b6167688cfcd25861ea4fd73ff0fb50856cfde42be9R17) and it is the [same string](https://github.com/zopefoundation/Products.Sessions/compare/4.9...4.11#diff-c6bb0b5b02771d839eae3ae7e8e1b71bcca5a744fb4bf3e72c0a9f72e08b41e8R13).